Source/WebKit/UIProcess/WebAuthentication/Cocoa/WebAuthenticationPanelClient.mm

```diff
@@ -53,7 +53,17 @@
     m_delegateMethods.panelRequestLAContextForUserVerificationCompletionHandler = [delegate respondsToSelector:@selector(panel:requestLAContextForUserVerificationWithCompletionHandler:)];
 }
 
-RetainPtr<id <_WKWebAuthenticationPanelDelegate> > WebAuthenticationPanelClient::delegate()
+void WebAuthenticationPanelClient::ref() const
+{
+    [delegate() retain];
```

Comment:
This still doesn't seem right? As far as I can tell, `WebAuthenticationPanelClient` is owned by the `WebAuthenticationPanel` not the `_WKWebAuthenticationPanelDelegate`. Also, `WebAuthenticationPanel`'s client may get overwritten via `setClient()`.